src/components/Footer.js

```diff
@@ -1,7 +1,21 @@
+import styled from 'styled-components';
+
 export default function Footer() {
 	return (
-		<footer>
-			<div></div>
-		</footer>
+		<StyledFooter>
+			<div>ᘻᗩᘜᓰᑢ ᘻᘿᗩᒪS</div>
+		</StyledFooter>
 	);
 }
+
+const StyledFooter = styled.div`
```

Comment:
If it is a footer why not use the `footer` element instead of a `div`? 🤔

Reply:
`return 
		<StyledFooter>
			<header>ᘻᗩᘜᓰᑢ ᘻᘿᗩᒪS</header>
		</StyledFooter>`

But then I would have a Header inside a Footer, still everything right?

Reply:
A `header` inside a `footer` is not ok. But a `h1` inside a `footer`is. [See Caniinclude.](https://caninclude.glitch.me/caninclude?child=h1&parent=footer)